Tests/SentryTests/Integrations/ANR/SentryANRTrackingIntegrationTests.swift

```diff
@@ -459,7 +513,42 @@ class SentryANRTrackingIntegrationTests: SentrySDKIntegrationTestsBase {
             XCTAssertEqual("crumb", breadcrumbs.first?.message)
         }
     }
-    
+
+    func testV2_ANRDetected_PauseCalledButStopNotCalled_SendsFatalANROnNextInstall() throws {
+        // Arrange
+        givenInitializedTracker(enableV2: true)
+        setUpThreadInspector()
+        Dynamic(sut).anrDetectedWithType(SentryANRType.nonFullyBlocking)
+        sut.pauseAppHangTracking()
+
+        // Act
+        givenInitializedTracker(enableV2: true)
+        sut.pauseAppHangTracking()
+
+        // Assert
+        try assertCrashEventWithScope { event, _ in
+            XCTAssertEqual(event?.level, SentryLevel.fatal)
+
+            let ex = try XCTUnwrap(event?.exceptions?.first)
+
+            XCTAssertEqual(ex.type, "Fatal App Hang Non Fully Blocked")
+            XCTAssertEqual(ex.value, "The user or the OS watchdog terminated your app while it blocked the main thread for at least 4500 ms.")
+
+            // We use the mechanism data to temporarily store the duration.
```

Comment:
`m`: why is there not stack trace assertion here?

Reply:
Cause we assert the stacktrace with other tests. Do you think we should also assert it here?

Reply:
If that's the case it's fine, wasn't clear to me from the name of the test.